temporal-sdk/src/main/java/io/temporal/client/UpdateWithStartWorkflowOperation.java

```diff
@@ -0,0 +1,413 @@
+    return new Builder<>(
+        () -> {
+          request.apply();
+        });
+  }
+
+  /**
+   * Returns a new builder for an {@link UpdateWithStartWorkflowOperation} for a one argument
+   * request.
+   *
+   * @param request method reference to a proxy created through {@link
+   *     WorkflowClient#newWorkflowStub(Class, WorkflowOptions)}.
+   * @param arg1 first request function parameter
+   */
+  public static <A1, R> Builder<R> newBuilder(Functions.Func1<A1, R> request, A1 arg1) {
+    return new Builder<>(() -> request.apply(arg1));
+  }
+
+  /**
+   * Returns a new builder for an {@link UpdateWithStartWorkflowOperation} for a two argument
+   * request.
+   *
+   * @param request method reference to a proxy created through {@link
+   *     WorkflowClient#newWorkflowStub(Class, WorkflowOptions)}.
+   * @param arg1 first request function parameter
+   * @param arg2 second request function parameter
+   */
+  public static <A1, A2, R> Builder<R> newBuilder(
+      Functions.Func2<A1, A2, R> request, A1 arg1, A2 arg2) {
+    return new Builder<>(() -> request.apply(arg1, arg2));
+  }
+
+  /**
+   * Returns a new builder for an {@link UpdateWithStartWorkflowOperation} for a three argument
+   * request.
+   *
+   * @param request method reference to a proxy created through {@link
+   *     WorkflowClient#newWorkflowStub(Class, WorkflowOptions)}.
+   * @param arg1 first request function parameter
+   * @param arg2 second request function parameter
+   * @param arg3 third request function parameter
+   */
+  public static <A1, A2, A3, R> Builder<R> newBuilder(
+      Functions.Func3<A1, A2, A3, R> request, A1 arg1, A2 arg2, A3 arg3) {
+    return new Builder<>(() -> request.apply(arg1, arg2, arg3));
+  }
+
+  /**
+   * Returns a new builder for an {@link UpdateWithStartWorkflowOperation} for a four argument
+   * request.
+   *
+   * @param request method reference to a proxy created through {@link
+   *     WorkflowClient#newWorkflowStub(Class, WorkflowOptions)}.
+   * @param arg1 first request function parameter
+   * @param arg2 second request function parameter
+   * @param arg3 third request function parameter
+   * @param arg4 fourth request function parameter
+   */
+  public static <A1, A2, A3, A4, R> Builder<R> newBuilder(
+      Functions.Func4<A1, A2, A3, A4, R> request, A1 arg1, A2 arg2, A3 arg3, A4 arg4) {
+    return new Builder<>(() -> request.apply(arg1, arg2, arg3, arg4));
+  }
+
+  /**
+   * Returns a new builder for an {@link UpdateWithStartWorkflowOperation} for a five argument
+   * request.
+   *
+   * @param request method reference to a proxy created through {@link
+   *     WorkflowClient#newWorkflowStub(Class, WorkflowOptions)}.
+   * @param arg1 first request function parameter
+   * @param arg2 second request function parameter
+   * @param arg3 third request function parameter
+   * @param arg4 fourth request function parameter
+   * @param arg5 sixth request function parameter
```

Comment:
```suggestion
   * @param arg5 fifth request function parameter
```

Reply:
I copied that from `BatchRequest` and didn't notice the issue. I'll fix it there, too 👍

Reply:
There were 16 places: https://github.com/temporalio/sdk-java/pull/2199/commits/3df72ba7e38e757d8f6d6209cdc8aee64ba9fc23